tests/ha/iscsi_client_setup.pm

```diff
@@ -28,10 +28,17 @@ sub run {
     ping_size_check(testapi::host_ip());
     ping_size_check($iscsi_server);
 
+    record_info 'iscsi initiator pre configuration', script_output('cat /etc/iscsi/initiatorname.iscsi', proceed_on_failure => 1);
+    record_info 'rpm-qf', script_output('rpm -qf /etc/iscsi/initiatorname.iscsi', proceed_on_failure => 1);
+
     # open-iscsi & iscsiuio
     zypper_call 'in open-iscsi' if (script_run('rpm -q open-iscsi'));
+    record_info 'iscsi initiator pre configuration', script_output('cat /etc/iscsi/initiatorname.iscsi', proceed_on_failure => 1);
```

Comment:
```suggestion
    record_info 'iscsi initiator pre configuration', 
     'Content:' . script_output('rpm -qf /etc/iscsi/initiatorname.iscsi', proceed_on_failure => 1) . "\nPackage:" . script_output('rpm -qf /etc/iscsi/initiatorname.iscsi', proceed_on_failure => 1);
```

Reply:
I plan to improve the code before I close the ticket in a subsequent PR, I will add this then.